integration_tests/http/raw-unsafe-path.yaml

```diff
@@ -50,6 +51,10 @@ requests:
       - |+
         GET /xyz/%u2s/%invalid HTTP/1.1
         Host: scanme.sh
+    # test relative path start with // (should not be removed)
+      - |+
+        GET //CFIDE/wizards/common/utils.cfc HTTP/1.1
```

Comment:
While this represent a nice addition also for `unsafe` requests. I think the integration test is specific for templates having `unsafe` flag not set. We can add a new integration test `raw-path.yaml` (if no equivalent exists already) where progressively track all these edge cases.

Reply:
yeah , @RamanaReddy0M we also have integration test will above cases with `unsafe: false` at https://github.com/projectdiscovery/nuclei/blob/main/integration_tests/http/http-paths.yaml

both in base and raw format